src/dex/aave-v3/aave-v3.ts

```diff
@@ -17,12 +17,16 @@ import { IDexHelper } from '../../dex-helper/idex-helper';
 import { Data, Param, PoolAndWethFunctions } from './types';
 import { SimpleExchange } from '../simple-exchange';
 import { Adapters, Config } from './config';
-import { getATokenIfAaveV3Pair } from './tokens';
+import { getATokenIfAaveV3Pair, setTokensOnNetwork } from './tokens';
 
 import WETH_GATEWAY_ABI from '../../abi/aave-v3-weth-gateway.json';
 import POOL_ABI from '../../abi/AaveV3_lending_pool.json';
+import { fetchTokenList } from './utils';
 
 const REF_CODE = 1;
+export const TOKEN_LIST_CACHE_KEY = 'token-list';
+const TOKEN_LIST_TTL_SECONDS = 86400; // 1 day
+const TOKEN_LIST_LOCAL_TTL_SECONDS = 10800; // 3 hours
```

Comment:
I think if you put it like: `24 * 60 * 60` and `3 * 60 * 60`, it is easier to validate

Reply:
Done.